### PR TITLE
Changed the assessmentReport file name to migration_assessment_report

### PR DESCRIPTION
**Pull request by priyanshi-yb**
* Also changed the `bulkAssessmentReport` file name to `bulk_assessment_report`.

Oracle Jenkins pipeline 
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3377/

### CODE REVIEW

File: yb-voyager/cmd/assessMigrationCommand.go [hunk 1]
```diff
@@ -478,7 +478,7 @@ func runAssessment() error {
 
 func checkStartCleanForAssessMigration(metadataDirPassedByUser bool) {
 	assessmentDir := filepath.Join(exportDir, "assessment")
-	reportsFilePattern := filepath.Join(assessmentDir, "reports", "assessmentReport.*")
+	reportsFilePattern := filepath.Join(assessmentDir, "reports", "migration_assessement_report.*")
```
Comment (makalaaneesh) [inline yb-voyager/cmd/assessMigrationCommand.go:481]: let's use this opportunity to declare a constant for the report name?

**Review by sanyamsinghal — COMMENTED** [review]
@priyanshi-yb can we change the report name for bulk cmd also in this PR.

**Comment by priyanshi-yb** [comment]
@sanyamsinghal , I am planning to do that.

**Review by sanyamsinghal — APPROVED** [review]
LGTM, thanks for this change.

**Pull request merged**